install/0000_00_cluster-version-operator_03_deployment.yaml

```diff
@@ -57,6 +57,7 @@ spec:
       nodeSelector:
         node-role.kubernetes.io/master: ""
       priorityClassName: "system-cluster-critical"
+      terminationGracePeriodSeconds: 130
```

Comment:
comment explaining where this value came from.

Reply:
Collapsing into https://github.com/openshift/cluster-version-operator/pull/424/#discussion_r466584923